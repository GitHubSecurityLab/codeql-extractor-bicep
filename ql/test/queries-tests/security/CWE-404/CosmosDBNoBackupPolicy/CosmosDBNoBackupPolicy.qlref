security/CWE-404/CosmosDBNoBackupPolicy.ql
