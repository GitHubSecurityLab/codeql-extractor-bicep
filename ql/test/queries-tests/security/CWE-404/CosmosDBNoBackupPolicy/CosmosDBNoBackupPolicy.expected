| app.bicep:4:1:14:1 | DatabaseResource[cosmosdb] | CosmosDB account 'vulnerable-cosmosdb-no-policy' does not have a backup policy configured, which may result in data loss. |
| app.bicep:17:1:24:1 | DatabaseResource[cosmosdb] | CosmosDB account 'vulnerable-cosmosdb-minimal' does not have a backup policy configured, which may result in data loss. |
